### PR TITLE
deployment: add gh-pages

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

add github pages

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -17,11 +17,16 @@ jobs:
...
+      - name: Setup node env 🏗
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14
+
       - name: Install dependencies 👨🏻‍💻
-        run: cd web & yarn
+        run: cd web && yarn
 
       - name: Build 🔧
```
Comment (alexanderbez) [inline .github/workflows/pages.yml:28]: We need to set `VUE_APP_ATLAS_API_ADDR = "https://api.atlas.cosmos.network/api/v1"` env var here when we build. This is how Atlas knows what API to reach out to.

Comment (marbar3778) [reply .github/workflows/pages.yml:28]: added, thanks

File: .github/workflows/pages.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Deployment
+# This job builds and deploys atlas to github pages.
+# It runs on every push to master.
+on:
+  push:
+    branches:
+      - master
```
Comment (alexanderbez) [inline .github/workflows/pages.yml:7]:
```suggestion
      - main
```

We don't use that verbiage here ;-)

**Review by alexanderbez — APPROVED** [review]
utACK

**Comment by marbar3778** [comment]
@helder-moreira do you have any pointers for getting this to work? It keeps erroring when i reload the page

**Comment by helder-moreira** [comment]
@marbar3778 do you mean reloads except on root?

**Comment by helder-moreira** [comment]
Looks like Github pages does not support SPA 😕 . Try looking at the workaround [here](https://medium.com/swlh/how-to-host-your-angular-reactjs-vuejs-spa-on-github-pages-2d9ab102ac7b)

**Comment by marbar3778** [comment]
Trying out your suggestion, but i cant build the front end anymore because scss does not support scss

**Comment by helder-moreira** [comment]
@marbar3778 why do you need arm64?

**Comment by marbar3778** [comment]
> @marbar3778 why do you need arm64?

My computer is arm based, new M1.

**Comment by helder-moreira** [comment]
> My computer is arm based, new M1.

Oh I see. Try checking if `docker` or `multipass` are already supported 🙂

**Pull request closed without merging**